### ISSUE TITLE
Unable to use the record wrapper on iPadOS

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/ph-scale#193]
For  phetsims/QA#514.
See https://github.com/phetsims/phet-io-wrappers/issues/363.

**Comment by pixelzoom** [comment]
Over in https://github.com/phetsims/phet-io-wrappers/issues/363#issuecomment-668244392, @kathy-phet identified this issue as non-blocking.   So closing this issue as "won't fix" for this release.
